### ISSUE TITLE
add mali gpu enhanced support

### ISSUE THREAD

**Issue by zay448345045** [longjunyu2/winlator#122]
https://github.com/Fcharan/WinlatorMali

**Comment by vedmysh** [comment]
What do you mean by "enhanced support"? The "major" thing they did is just removed dxvk and turnip and put some software distribution inside the winlator :) Why dont just use that fork on mali?

**Comment by zay448345045** [comment]
> What do you mean by "enhanced support"? The "major" thing they did is just removed dxvk and turnip and put some software distribution inside the winlator :) Why dont just use that fork on mali?

Yes you are right.It seems that the author in charge of editing is quite radical. I mean my idea is to add software emulation only in the GPU emulation option without deleting it. Indeed, because MediaTek's drivers are closed source.Like Disable Threaded Presentation.Because Stops the use of a background thread for presenting frames when using the Vulkan renderer,It will help improve the simulation performance of using Mali series GPU devices.
